helm/cluster/templates/clusterapi/controlplane/kubeadmcontrolplane.yaml

```diff
@@ -51,4 +55,6 @@ spec:
     {{- include "cluster.internal.controlPlane.kubeadm.postKubeadmCommands" $ | indent 4 }}
     users:
     {{- include "cluster.internal.kubeadm.users" $ | indent 4 }}
+  replicas: {{ .Values.global.controlPlane.replicas }}
+  version: v{{ trimPrefix "v" .Values.internal.kubernetesVersion }}
```

Comment:
why do we have to trim the prefix `v` and then adding it again?

Reply:
I think it's in order to support `.Values.internal.kubernetesVersion` to both have and not have the `v` prefix, i.e. we don't care if the prefix is there, because we "clean up" the version from potential prefix and then we ensure it's there.

Took it from cluster-aws here https://github.com/giantswarm/cluster-aws/blob/acf1b9c712a51b364f5bece1481b3b3e3efdc263/helm/cluster-aws/templates/_control_plane.tpl#L247.